### PR TITLE
docs: improve authorization guide

### PR DESCRIPTION
**Pull request by bastienbeurier**
- Fix docs redirects
- Improve authorization guide

### CODE REVIEW

File: docs-v2/guides/authorize-an-api-from-your-app-with-custom-ui.mdx [hunk 1]
```diff
@@ -4,20 +4,20 @@ sidebarTitle: 'Authorize an API from you app (custom UI)'
...
+Pre-requisites: 
+- complete the [_Configure an integration_ guide](/guides/getting-started/configure-an-integration)
+- generate a [Connect session token](/guides/getting-started/authorize-an-api-from-your-app#generate-a-session-token-backend)
+</Info>
 
 ## Integrate the frontend SDK
 
-Once you have tested that the authorization flow works for your own external account, you can trigger authorization flows for your customers from your app with the Nango SDK.
-
-Get your `Public Key` from the _Environment Settings_ tab.
```
Comment (bodinsamuel) [inline docs-v2/guides/authorize-an-api-from-your-app-with-custom-ui.mdx:13]: Thanks I saw this leftover yesterday 🙏🏻

File: docs-v2/guides/getting-started/authorize-an-api-from-your-app.mdx [hunk 2]
```diff
@@ -43,79 +70,63 @@ api.post('/sessionToken', (req, res) => {
...
-api.post('/connection', (req, res) => {
-  await User.update({
-    connectionId: req.body.connectionId,
-    integrationId: req.body.providerConfigKey,
-  });
-
-  res.status(200).send({
-    success: true
-  });
-});
```
Comment (bodinsamuel) [inline docs-v2/guides/getting-started/authorize-an-api-from-your-app.mdx:105]: The reason I added this code snippet is that it's impossible to use webhook right now with a session token, you'll receive a random connectionId with no link to anything

**Review by bodinsamuel — APPROVED** [review]
Looks good, however I believe removing the save connectionId snippet is not a good idea rn.

File: docs-v2/guides/getting-started/authorize-an-api-from-your-app.mdx (hunk 2 above)
Comment (bastienbeurier) [reply docs-v2/guides/getting-started/authorize-an-api-from-your-app.mdx:105]: This should be fix thanks to your change, and the docs are updated.

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**